test/WebSites/BasicWebSite/Filters/GlobalActionFilter.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace BasicWebSite
+{
+    public class GlobalActionFilter : IActionFilter
+    {
+        public void OnActionExecuted(ActionExecutedContext context)
+        {
+            if (context.ActionDescriptor.DisplayName ==
+                "BasicWebSite.Controllers.HomeController.GetTextFromFilterAddedByOptions")
+            {
+                context.Result = new ContentResult()
+                {
+                    Content = "This was added by filter.",
+                    ContentType = "text/plain"
+                };
+            }
+        }
+
+        public void OnActionExecuting(ActionExecutingContext context)
+        {
+        }
+    }
```

Comment:
Why not use ApplicationModel to add this to the action that you're interested in. These 'global' filters that only act on one action are a bad example for users and are hard to debug.